SU-SWS/decanter
# V6 haiwr 266 pull quote

# READY FOR advice


# Summary
This adds pull-quote functionality

# Needed By (Date)


# Urgency
not urgent

# Steps to Test

1. Add html to text area
1. Verify the pull quotes display as expected

```
  <h2>Pull-quote</h2> 

  <p>Fusce a quam. Nunc nec neque. In consectetuer turpis ut velit. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Nunc interdum lacus sit amet orci.</p>

<aside class="su-pullquote-left">
<p>This is the pull-quote text. Pellentesque dapibus hendrerit tortor. Nullam quis ante. Proin viverra, ligula sit amet ultrices semper, ligula arcu tristique sapien, a accumsan nisi mauris ac eros.</p>
</aside>

<p>Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum. Etiam sollicitudin, ipsum eu pulvinar rutrum, tellus ipsum laoreet sapien, quis venenatis ante odio sit amet eros.</p>

<p>Praesent ac massa at ligula laoreet iaculis. Phasellus dolor. Proin faucibus arcu quis ante. Donec sodales sagittis magna. Nullam accumsan lorem in dui.</p>

<p>In dui magna, posuere eget, vestibulum et, tempor auctor, justo. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Suspendisse nisl elit, rhoncus eget, elementum ac, condimentum eget, diam. Curabitur blandit mollis lacus. Morbi mattis ullamcorper velit.</p>

<p>Donec id justo. Nullam nulla eros, ultricies sit amet, nonummy id, imperdiet feugiat, pede. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Aenean imperdiet.</p>
 <p>Fusce a quam. Nunc nec neque. In consectetuer turpis ut velit. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Nunc interdum lacus sit amet orci.</p>

<aside class="su-pullquote-right">
<p>This is pull quote text. Pellentesque dapibus hendrerit tortor. Nullam quis ante. Proin viverra, ligula sit amet ultrices semper, ligula arcu tristique sapien, a accumsan nisi mauris ac eros.</p>
</aside>

<p>Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero, sit amet adipiscing sem neque sed ipsum. Etiam sollicitudin, ipsum eu pulvinar rutrum, tellus ipsum laoreet sapien, quis venenatis ante odio sit amet eros.</p>

<p>Praesent ac massa at ligula laoreet iaculis. Phasellus dolor. Proin faucibus arcu quis ante. Donec sodales sagittis magna. Nullam accumsan lorem in dui.</p>

<p>In dui magna, posuere eget, vestibulum et, tempor auctor, justo. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Suspendisse nisl elit, rhoncus eget, elementum ac, condimentum eget, diam. Curabitur blandit mollis lacus. Morbi mattis ullamcorper velit.</p>

<p>Donec id justo. Nullam nulla eros, ultricies sit amet, nonummy id, imperdiet feugiat, pede. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Aenean imperdiet.</p>

</div>
```

# Affected Projects or Products


# Associated Issues and/or People
- JIRA ticket HAIWR-266
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
